system/system_monitor/src/net_monitor/net_monitor.cpp

```diff
@@ -454,7 +452,6 @@ void NetMonitor::update_network_capacity(NetworkInfomation & network, int socket
   strncpy(request.ifr_name, network.interface_name.c_str(), IFNAMSIZ - 1);
   request.ifr_data = (caddr_t)&ether_request;  // NOLINT [cppcoreguidelines-pro-type-cstyle-cast]
 
-  ether_request.cmd = ETHTOOL_GSET;
```

Comment:
@veqcc 
Casting `ether_request` to `request.ifr_data`, so `ether_request.cmd` is used and passing `request` to `ioctl`.

Reply:
@ito-san 
Thank you for your quick reply!!
Then, do you have any idea to avoid cppcheck warning, without using suppression comments?

Reply:
@veqcc Yes! it's worth trying that approach.